q2cli/handlers.py

```diff
@@ -336,12 +336,12 @@ def get_click_options(self):
             name = '--' + self.cli_name
 
         if self.default is NoDefault:
-            yield click.Option([name], type=type, help='[required]')
```

Comment:
We still want to specify the type for non-boolean types. I think you'll need to special-case boolean here